packages/js/product-editor/src/blocks/regular-price/edit.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { Link } from '@woocommerce/components';
+import { CurrencyContext } from '@woocommerce/currency';
+import { getNewPath } from '@woocommerce/navigation';
+import { recordEvent } from '@woocommerce/tracks';
+import { useBlockProps } from '@wordpress/block-editor';
+import { BlockEditProps } from '@wordpress/blocks';
+import { useInstanceId } from '@wordpress/compose';
+import { useEntityProp } from '@wordpress/core-data';
+import {
+	createElement,
+	useContext,
+	createInterpolateElement,
+} from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	BaseControl,
+	// @ts-expect-error `__experimentalInputControl` does exist.
+	__experimentalInputControl as InputControl,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useCurrencyInputProps } from '../../hooks/use-currency-input-props';
+import { formatCurrencyDisplayValue } from '../../utils';
+import { SalePriceBlockAttributes } from './types';
+import { useValidation } from '../../hooks/use-validation';
+
+export function Edit( {
+	attributes,
+}: BlockEditProps< SalePriceBlockAttributes > ) {
+	const blockProps = useBlockProps();
+	const { label, help } = attributes;
+	const [ regularPrice, setRegularPrice ] = useEntityProp< string >(
+		'postType',
+		'product',
+		'regular_price'
+	);
+	const [ salePrice ] = useEntityProp< string >(
+		'postType',
+		'product',
+		'sale_price'
+	);
+	const context = useContext( CurrencyContext );
+	const { getCurrencyConfig, formatAmount } = context;
+	const currencyConfig = getCurrencyConfig();
+	const inputProps = useCurrencyInputProps( {
+		value: regularPrice,
+		setValue: setRegularPrice,
+	} );
+
+	const interpolatedHelp = help
```

Comment:
I don't see `help` currently being used.  Do we have plans to use this in the future?

Reply:
Yes it is. See the next [line](https://github.com/woocommerce/woocommerce/pull/38078/files/179d419e93a7c7e5eedd1eaefc16b43f204662f3#diff-22543645991a77248897b85d0248931df6f38316cd7de38c2245972961b50316R57)

Reply:
I can see it used in this component, but was curious if this was being actively passed as an attribute to this component.

I see now that a `help` attribute is being passed in [the PR ](https://github.com/woocommerce/woocommerce/pull/37961)merged in last week so this looks good. 👍